src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+
+  if (days >= 3) {
+    if (days >= 7) {
+      return (cost - 50);
+    }
+
+    return (cost - 20);
```

Comment:
```suggestion
    return cost - 20;
```

Reply:
in case you don't do any different calculations, you don't need extra parentheses